packages/test-app-frontend/vite.config.ts

```diff
@@ -69,12 +43,13 @@ export default defineConfig(() => ({
   },
 }));
 
-class StringReplacePlugin implements Plugin {
-  public name = StringReplacePlugin.name;
-  public enforce = "pre" as const;
-
-  public transform(code: string): string {
-    // iTwin.js by default injects a font that is incorrect and lacks some required font weights
-    return code.replace("document.head.prepend(openSans);", "// document.head.prepend(openSans);");
-  }
+function stringReplacePlugin(): Plugin {
+  return {
+    name: stringReplacePlugin.name,
+    enforce: "pre",
+    transform: (code: string) => {
+      // iTwin.js by default injects a font that is incorrect and lacks some required font weights
```

Comment:
Have you created an issue against their repositories or followed up regarding this with core?

Reply:
I have raised this issue within iTwin.js team three years ago but this font injection code is still present in [@itwin/core-frontend](https://github.com/iTwin/itwinjs-core/blob/7bdb8cb11bbbe9a3f7fe0c9eb0d3f7191ba48934/core/frontend/src/IModeljs-css.ts#L155).

The issue however, nowadays is dormant; all iTwinUI3 applications use NotoSans, and web browser never requests OpenSans font.